apps/finance/app/src/components/Balances.js

```diff
@@ -80,88 +80,89 @@ function useBalanceItems(balances) {
 
 function Balances({ balances, compactMode }) {
   const theme = useTheme()
+  const { layoutName } = useLayout()
```

Comment:
ahh now we use `layoutName` instead of `compact = bellow(...)` ?

Reply:
We can yes, but I am thinking about merging the two because it’s a bit confusing.